### PR TITLE
Make the copyright year automatically update.

### PR DESCRIPTION
**Pull request by e3ndr**
Current value is almost two years out-of-date. I think it's time we got it updated :^)

### CODE REVIEW

**Comment by datagutt** [comment]
Ender touching a PHP project

**Comment by e3ndr** [comment]
> 

i'm crying and shaking rn. php has taken my family.

**Comment by vasilvestre** [comment]
> Ender touching a PHP project

It's fine, it's only some twig template so.. he's safe. I want to test the changes before merge then we're fine :)

File: templates/footer.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <footer class="items-center p-4 footer bg-neutral text-neutral-content flex fixed bottom-0 justify-between w-10/12 self-center bg-gray-100">
     <div class="flex items-center">
         <img src="{{ asset('favicon.svg') }}" width="24" height="24"/>
-        <p class="ml-2">Copyright © 2021 - All right reserved</p>
+        <p class="ml-2">Copyright © {{ 'now' | date('Y') }} - All right reserved</p>
```
Comment (artandor) [inline templates/footer.html.twig]: Hi, thanks for your PR

I believe the copyright should be "2021 - current" date instead of just the current date.

Comment (e3ndr) [reply templates/footer.html.twig]: According to my light Googling, this solution should retrieve just the current year and insert it into the rendered template.

Comment (artandor) [reply templates/footer.html.twig]: Yeah this should work, but the rendered result should be like the following imo :

Copyright © 2021 - 2022 - All right reserved

Not just Copyright © 2022 - All right reserved

I believe most Copyright are showing the start year aswell as the current

Comment (e3ndr) [reply templates/footer.html.twig]: Yeah I completely misread your comment. Guess I shouldn't check email early in the morning 😅.

**Pull request closed without merging**